scripts/check-translations.ps1

```diff
@@ -0,0 +1,83 @@
+# Step 1: Find all instances of vscode.l10n.t() and extract the strings from .ts and .tsx files
+$withParamsPattern = 'vscode\.l10n\.t\(["' + "`'" + '`](.+?)["' + "`'" + '`],'
```

Comment:
```suggestion
$withParamsPattern = 'vscode\.l10n\.t\(["''`](.+?)["''`],'
```

Won't this regex be easier to read as a continuous string? You can escape a single quote by prefixing it with a single quote.

See [these docs](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_quoting_rules?view=powershell-7.4)

Reply:
Yeah... committing this as they both return the same result